cmd/podsync/updater.go

```diff
@@ -257,6 +258,10 @@ func (u *Updater) fileSize(path string) (int64, error) {
 	return info.Size(), nil
 }
 
+func (u *Updater) episodeNameOutputTemplate(episode *model.Episode) string {
+	return fmt.Sprintf("%s.%s", episode.ID, "%(ext)s")
```

Comment:
Can we move this under `YoutubeDl`, to keep youtube-dl specific code in one place behind `Downloader` interface.

Reply:
Makes sense.